### PR TITLE
feat: Added the SimpleCalc, Counter, TubePlayer and ToDo apps to the ReadME

### PR DESCRIPTION
**Pull request by vatsashah45**
…ReadME

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
+
+The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using 4 options - [C# Markup](https://aka.platform.uno/csharp-markup) or [XAML]() for the UI and [MVUX](https://aka.platform.uno/mvux) or [MVVM]() for state management.
```
Comment (kazo0) [inline README.md:36]: Missing links for `[XAML]()` and `[MVVM]()`?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+
+### ToDo App
+
+Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)
+
+### TubePlayer App
+
+The TubePlayer  App is a sample application that leverages the Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers looking to understand how to use Uno.Extensions to create a rich, responsive video streaming service.
```
Comment (kazo0) [inline README.md:54]:
```suggestion
The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers looking to create a rich, responsive video streaming service.
```

Comment (kazo0) [inline README.md:54]: `to create a rich, responsive video streaming service.`

I'm not sure that TubePlayer's UI is responsive, is it?

File: README.md (hunk 1 above)
Comment (kazo0) [reply README.md:36]: For these links I would point directly to the variants in the workshop tutorial docs:

image

I'm not sure we have aka links created for these though @agneszitte

File: README.md (hunk 2 above)
Comment (kazo0) [reply README.md]: @agneszitte did we change TubePlayer to be responsive? I thought it was locked to a fixed width or something like that?

File: README.md [hunk 3]
```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
 
+### SimpleCalculator App
+
+The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using 4 options - [C# Markup](https://aka.platform.uno/csharp-markup) or [XAML]() for the UI and [MVUX](https://aka.platform.uno/mvux) or [MVVM]() for state management.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)
+
+### Counter App
+
+Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.
```
Comment (agneszitte) [inline README.md:42]: @vatsashah45, @kazo0, @nickrandolph 
I think we could use this simple aka link (https://aka.platform.uno/counter-tutorial) that gives the overview of the four variants of the Counter workshop.

So we can mention something additional like:
`This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).`

File: README.md (hunk 1 above)
Comment (agneszitte) [reply README.md:36]: @vatsashah45, @kazo0, @nickrandolph
I think we could use this simple aka link (https://aka.platform.uno/simplecalc-workshop) that gives the overview of the four variants of the SimpleCalc workshop.

So we can mention something additional like:
`The sample app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).`

File: README.md [hunk 4]
```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+
+### ToDo App
+
+Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)
+
+### TubePlayer App
+
+The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers to create a rich, responsive video streaming service.
```
Comment (agneszitte) [inline README.md:54]: @vatsashah45, @kazo0, @nickrandolph 
I think we could use this simple aka link (https://aka.platform.uno/tubeplayer-workshop) that gives the overview of the Tube Player workshop.

So we can mention something additional like:
`This sample app was built following the [Tube Player worshop](https://aka.platform.uno/tubeplayer-workshop).`

File: README.md (hunk 2 above)
Comment (agneszitte) [reply README.md]: @kazo0 yes still locked to a fixed width. You are right we need to remove the "responsive" part here as that is not an aspect we are showcasing with this sample.

@kazo0, @nickrandolph 
I think the summary of this sample should be closer to some details we can find in the workshop overview, right ?
image

File: README.md [hunk 5]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
```
Comment (agneszitte) [inline README.md:34]:
```suggestion
### Simple Calc App
```
To match the workshop and name of the sample app

File: README.md [hunk 6]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+
+### ToDo App
+
+Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)
+
+### TubePlayer App
+
+The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers to create a rich video streaming service.
```
Comment (agneszitte) [inline README.md:55]: @kazo0, @nickrandolph do you think "The TubePlayer App is a sample application that leverages Uno Platform **to recreate the core functionalities of YouTube**" isn't a bit much here? 

As it is just a search and stream video experience.
image

File: README.md (hunk 5 above)
Comment (lukeblevins) [reply README.md:34]: Also try to update any subsequent uses of _SimpleCalculator_

**Review by lukeblevins — COMMENTED** [review]
Good changes to the README that highlight flagship samples from our workshops. 

Just minor feedback from me that matches up with that from @agneszitte

**Review by agneszitte — CHANGES_REQUESTED** [review]
This PR https://github.com/unoplatform/Uno.Samples/pull/643 will need to be merged first before merging these changes for the links to the sources to properly work for Counter and ToDo

File: README.md [hunk 7]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalc App
+
+The SimpleCalc App is a sample application that is designed to perform basic arithmetic operations. The app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
```
Comment (Xiaoy312) [inline README.md:36]:
```suggestion
The SimpleCalc App is a sample application designed to perform basic arithmetic operations. This app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining different markup languages (XAML or C# Markup) and presentation frameworks (MVVM or MVUX).
```

File: README.md [hunk 8]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)
+
+### Counter App
+
+Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.
```
Comment (Xiaoy312) [inline README.md:42]:
```suggestion
Experience the simplicity and power of Uno.Extensions through the Counter App, a straightforward yet powerful demonstration of both basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and real-time UI updates.
```

File: README.md [hunk 9]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter)
+
+### ToDo App
+
+Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.
```
Comment (Xiaoy312) [inline README.md:49]:
```suggestion
Dive into the essentials of task management with the ToDo App, a meticulously crafted sample application that highlights the power and flexibility of Uno.Extensions. By emphasizing the creation and organization of to-do lists, this app showcases practical applications of essential Uno.Extensions features, offering a hands-on experience in crafting responsive and user-friendly interfaces across multiple platforms.
```

File: README.md [hunk 10]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)
+
+### TubePlayer App
+
+The TubePlayer App is a sample application that enables the user to search for, and stream, Youtube videos. This app was made using tools, libraries, and patterns available in the Uno Platform, that are there to help rapidly build high-quality applications.
```
Comment (Xiaoy312) [inline README.md:55]:
```suggestion
The TubePlayer App is a sample application that allows users to search for, and stream YouTube videos. This app was created using the tools, libraries, and patterns provided by the Uno Platform, designed to facilitate the rapid development of high-quality applications.
```

File: README.md [hunk 11]
```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
...
+This sample app was built following the [Tube Player worshop](https://aka.platform.uno/tubeplayer-workshop).
```
Comment (Xiaoy312) [inline README.md:56]:
```suggestion
This sample app was built following the [Tube Player workshop](https://aka.platform.uno/tubeplayer-workshop).
```

**Review by Xiaoy312 — APPROVED** [review]

**Review by kazo0 — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

**Pull request merged**